demos/github-demo/src/isograph-components/__isograph/Query/PullRequest/raw_response_type.ts

```diff
@@ -0,0 +1,29 @@
+export type Query__PullRequest__rawResponse = {
+  repository____owner___v_repositoryOwner____name___v_repositoryName: {
+    id: string,
+    pullRequest____number___v_pullRequestNumber: {
+      id: string,
+      bodyHTML: string,
+      comments____last___l_10: {
+        edges: {
+          node: {
+            id: string,
+            author: {
+              __typename: string,
```

Comment:
🤔  can we make this a union of concrete strings of types that implement `Actor`? Not a blocker